defaults/main.yml

```diff
@@ -27,7 +27,8 @@ roundcube__nginx_access_policy: ''
 # List of PHP5 packages required by Roundcube.
 roundcube__php5_packages: [ 'php-auth-sasl', 'php5-gd', 'php5-intl', 'php5-json',
                            'php5-mcrypt', 'php-mail-mime', 'php-mail-mimedecode',
-                           'php-net-smtp', 'php-net-socket', 'php-pear' ]
+                           'php-net-smtp', 'php-net-socket', 'php-pear',
+                           'php-net-idna2', 'php-net-ldap3' ]
```

Comment:
It's not an unreasonable request, I think. Current LDAP support in DebOps is very basic. There should be a set of Ansible local facts that indicate that the host is part of the LDAP domain and where to find the server, etc. The `roundcube` role could check if a particular fact related to LDAP exists and act accordingly. The specific set of LDAP packages would need to be in a separate variable for this to work conditionally.

I'd say, leave the current situation as-is (`php-net-ldap3` package not installed by default) and when the LDAP support in DebOps improves, we can add that back.